data/json/items/book/traps.json

```diff
@@ -52,9 +52,8 @@
   {
     "id": "textbook_traps",
     "type": "BOOK",
-    "name": "The Compleat Trapper",
-    "name_plural": "The Compleat Trapper",
-    "//": "The spelling is intentionally archaic, as this form is commonly used in the titles of books.",
```

Comment:
Right but because this book covers modern techniques it shouldn't have the old spelling. If this was a purely old style book it would be fine. Plus all the other "Complete X" books use the modern spelling.